opentracing-api/src/main/java/io/opentracing/ScopeManager.java

```diff
@@ -26,15 +26,28 @@
 public interface ScopeManager {
 
     /**
+     * @deprecated use {@link #activate} instead.
```

Comment:
nit move to the bottom and say when it will be removed (ideally next release as would the other deprecated methods added prior.. deprecation pre 1.0 is a strange thing but anyway)

Reply:
I don't think there's a date to remove this one (we should discuss this topic, but another time ;) ), but we could say "in the next releases" (and definitely before 1.0 ;) )